spec/integration_spec.rb

```diff
@@ -411,10 +434,27 @@
       end
     end
 
-    context 'for not enabled percentage of time' do
+    context 'for enabled float percentage of time' do
       before do
         # ensure percentage of time returns percentage that makes five percent
```

Comment:
super minor, but should this now say *that makes 4.1 percent* instead of five since its being enabled for 4.1% now